src/views/registration-js/attendees/fields/datetime.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * This template renders a Single Ticket content
+ * composed by Title and Description currently
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/registration-js/attendees/fields/datetime.php
+ *
+ * @version TBD
+ *
+ * @see Tribe__Tickets_Plus__Meta__Field__Datetime
+ *
+ */
+$required   = isset( $field->required ) && 'on' === $field->required ? true : false;
+$option_id  = "tribe-tickets-meta_{$field->slug}_{$ticket->ID}{{data.attendee_id}}";
+$field      = (array) $field;
+$field_name = 'tribe-tickets-meta[' . $ticket->ID . '][{{data.attendee_id}}][' . esc_attr( $field['slug'] ) . ']';
+$disabled   = false;
+$classes    = [ 'tribe-common-b1', 'tribe-field', 'tribe-tickets__item__attendee__field__datetime' ];
+if ( $required ) {
+	$classes[] = 'tribe-tickets-meta-required';
+}
+?>
+<div <?php tribe_classes( $classes ); ?> >
+	<label
+			class="tribe-common-b2--min-medium tribe-tickets-meta-label"
+			for="<?php echo esc_attr( $option_id ); ?>"
+	><?php echo wp_kses_post( $field['label'] ); ?><?php tribe_required_label( $required ); ?></label>
+	<input
+			type="date"
+			id="<?php echo esc_attr( $option_id ); ?>"
+			class="tribe-common-form-control-datetime__input ticket-meta"
+			name="<?php echo esc_attr( $field_name ); ?>"
+			value="<?php echo esc_attr( $value ); ?>"
+			min="<?php echo date( 'Y' ) - 100 ?>-01-01"
+			max="<?php echo date( 'Y' ) + 100 ?>-31-12"
+			<?php tribe_required( $required ); ?>
```

Comment:
* :no_entry_sign: Inline PHP statement must end with a semicolon
* :no_entry_sign: All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found 'date'.
* :no_entry_sign: date() is affected by runtime timezone changes which can cause date/time to be incorrectly displayed. Use gmdate() instead.
* :warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not ')'

<sub>hash: 658e8b7b383b49c7c10660045f3c11d7</sub>